router/dataplane_spec.gobra

```diff
@@ -20,9 +20,15 @@ import (
 	"net"
 	"hash"
 	sl "github.com/scionproto/scion/verification/utils/slices"
+	"github.com/scionproto/scion/verification/io"
```

Comment:
The spacing in the file seems to be inconsistent. Please address it everywhere. In particular, do not have two or more consecutive empty lines, and do not have two or more consecutive spaces together.

Reply:
I removed all consecutive spaces except the ones in the predicate. Should I remove those as well?